src/classes/challenge.py

```diff
@@ -26,7 +26,8 @@ def parse_rootme_challenge_data(data):
         author_id = int(main_author["id_auteur"])
 
         title = data["titre"]
-        category = data["rubrique"]
+        # parse the challenge category to ASCII character and better usability
+        category = data["rubrique"].lower().replace(" ", "").replace('\u00E9', 'e').replace('\u00E8', 'e')
```

Comment:
If it is a string containing '\u' you could do a kind of trick with a `'\u00E9\u00E8'.encode('utf-8').decode('utf-8')` which prints the correct characters, is more generic to other unicode char i think (you could just add option to the encode and decode method to handle when the unicode char can't be transposed to utf-8), and I think utf-8 should be handle correctly by the rest of the code (i check SQLite for example) and I suppose basic python string can print chars like 'é' and 'è'.
Otherwise you could check if the type is not directly unicode string `u"string"` or byte array.

Reply:
Oh no it's not the pb actually! 
It's just that RootMe API returns challenge category such as `App - Système`. At first my pb was to match them with your asset file name `app-systeme.png` so I did this dirty parsing in the `NewValidatedChallenge.make_category` method but I thought that anyway `app-systeme` was far easier to work with than `App - Système` so I decided to store them without blank characters, in lowercase, and ASCII only! 
However you're right I could write `.replace('é', 'e')` but I don't know I used '\u' format 🙃